DolbyIO/Source/Private/DolbyIOSubsystem.cpp

```diff
@@ -255,29 +283,37 @@ void UDolbyIOSubsystem::Initialize(const FString& Token)
 	        [this](dolbyio::comms::event_handler_id)
 	        {
 		        return Sdk->conference().add_event_handler(
-		            [this](const dolbyio::comms::participant_updated& Event)
+		            [this](const dolbyio::comms::participant_added& Event)
 		            {
-			            if (!Event.participant.status)
+			            if (!Event.participant.status ||
+			                LocalParticipantID == FString(Event.participant.user_id.c_str()))
```

Comment:
`Event.participant.user_id == ToStdString(LocalParticipantID)`

Reply:
as you wish :)